index.html

```diff
@@ -289,11 +259,56 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-    </main>
 
-    <script
-      src="https://kit.fontawesome.com/e966a3bd22.js"
-      crossorigin="anonymous"
-    ></script>
+      <footer id="Contact">
+        <section class="footer-head">
+          <h2>
+            I'm always <br />
+            interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </h2>
+        </section>
+
+        <form class="footer-form" action="https://formspree.io/f/xdobnpad" method="post">
+          <input type="text" id="name" name="user-name" placeholder="Full Name" required />
+          <input type="text" id="email" name="user-email" placeholder="Email Address" required />
+          <textarea
```

Comment:
## Changes Required!

⚠️ Your form looks pretty good. However, it is one of the project requirements that you add a `maxlength="30"` attribute to the name input, and a `maxlength="500"` attribute to the email input. Please kindly add them!

Additionally, your email input should be of `type="email"` ✉️